tetris.py

```diff
@@ -205,11 +208,11 @@ def draw_rectangle(self, x, y, tags, fill):
             fill=fill,
         )
 
-    def draw_block(self):
+    def draw_block(self, block, block_letter, landed_blocks):
```

Comment:
`landed_blocks` not needed for `small_board`, so add handy default:

```suggestion
    def draw_block(self, block, block_letter, landed_blocks=None):
```